<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaagenKalibrierung</class>
 <widget class="QWidget" name="WaagenKalibrierung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Futterkarre - Waagenkalibrierung</string>
  </property>
  <widget class="QLabel" name="lbl_header">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>WAAGENKALIBRIERUNG</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ZURÜCK</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_gesamtgewicht_titel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GESAMTGEWICHT:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_gesamtgewicht_wert">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string>border: 2px solid #666; padding: 10px;</string>
   </property>
   <property name="text">
    <string>0.00 kg</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_einzelzellen_titel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>EINZELZELLEN:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_hx711_0_status">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>250</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>HX711 VL Status</string>
   </property>
   <property name="styleSheet">
    <string>background-color: #ffffff; border: 2px solid #cccccc; border-radius: 5px; text-align: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_vl_titel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>255</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Vorne Links:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_vl_wert">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>255</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.00 kg</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_hx711_1_status">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>295</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>HX711 VR Status</string>
   </property>
   <property name="styleSheet">
    <string>background-color: #ffffff; border: 2px solid #cccccc; border-radius: 5px; text-align: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_vr_titel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>300</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Vorne Rechts:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_vr_wert">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>300</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.00 kg</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_hx711_2_status">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>250</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>HX711 HL Status</string>
   </property>
   <property name="styleSheet">
    <string>background-color: #ffffff; border: 2px solid #cccccc; border-radius: 5px; text-align: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_hl_titel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>255</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Hinten Links:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_hl_wert">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>255</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.00 kg</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_hx711_3_status">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>295</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>HX711 HR Status</string>
   </property>
   <property name="styleSheet">
    <string>background-color: #ffffff; border: 2px solid #cccccc; border-radius: 5px; text-align: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_hr_titel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>300</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Hinten Rechts:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_hr_wert">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>300</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.00 kg</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_schritt_titel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>KALIBRIERUNG:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_schritt1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1. Karren komplett leeren</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tara">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>420</y>
     <width>181</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TARA</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_schritt2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>490</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>2. Referenzgewicht auflegen:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_referenzgewicht">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>485</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>20.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_kg">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>490</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>kg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_kalibrieren">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>480</y>
     <width>181</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>KALIBRIEREN</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_schritt3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>3. Kalibrierung testen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_test">
   <property name="geometry">
    <rect>
     <x>449</x>
     <y>540</y>
     <width>181</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TEST</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_status_titel">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>380</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>STATUS:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_status">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>430</y>
     <width>500</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string>border: 1px solid #666; padding: 8px;</string>
   </property>
   <property name="text">
    <string>Bereit für Kalibrierung...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_speichern">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>540</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SPEICHERN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>540</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_live_update">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>620</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Live-Anzeige (1/s)</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_toleranz">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>620</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Toleranz:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_toleranz">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>615</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>0.05</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_toleranz_einheit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>620</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Sans Fallback</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>kg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../Futterwagen/Futterkarre/icons/icons.qrc">
     <normaloff>:/icons/Back.png</normaloff>:/icons/Back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../Futterwagen/Futterkarre/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
